operations/boot_orchestration/Tools_for_Resolving_Boot_Issues.md

```diff
@@ -2,6 +2,27 @@
 
 A number of tools can be used to analyze and debug issues encountered during the compute node boot process. The underlying issue and symptoms dictate the type of tool required.
 
+## BOS
+
+BOS component records offer a history of operations that have been applied to a node.
+
+```bash
+cray bos components describe x3000c0s19b4n0
+```
+
+Additionally, nodes that are unable to boot may also be disabled. BOS will not apply power actions to disabled nodes,
```

Comment:
Is there any other BOS operation that can clear the "enabled = False" state, or does the administrator have sole control over that?

Reply:
At the end of the day, the component record is served up by an API, so any software or user could change this, however, in practice, the only things that toggle the enabled/disabled flag on the BOS level are:
- Nodes completing boot will toggle from enabled to disabled
- New BOS boot sessions that include the affected components will move from disabled to enabled
- Any other user who decides to enable or disable operation on a given BOS component.

Reply:
@don-bahls-hpe , note that in this case enabled = True means that BOS is going to operate on it. enabled = False means BOS is not going to operate on it. But this is a dynamic state. Nodes are enabled when a new BOS session is created. That is the first step.